backend/src/openbeheer/zaaktype/utils.py

```diff
@@ -0,0 +1,15 @@
+from openbeheer.types import ZGWError
+
+
+def format_related_resource_error(
+    field_name: str, error: ZGWError, index: int
+) -> ZGWError:
+    if not hasattr(error, "invalid_params") or not isinstance(
+        error.invalid_params, list
+    ):
+        return error
```

Comment:
`bool(UNSET) == False` so we could write
```suggestion
    if not getattr(error, "invalid_params", None):
        return error
```